README.md

```diff
@@ -32,6 +32,32 @@ Run `yarn start` and Wire's web app will be available at: http://localhost:8080/
 
 To login with your existing Wire account use: http://localhost:8080/auth/?env=prod#login
 
+### Testing
+
+To launch the full test suite (types check + linting + server tests + app tests), simply run
+
+`yarn test`
+
+Alternatively, you can run specific parts of the app
+
+`yarn test:(server|types|auth|app)`
+
+Since the test suite for the app is the biggest test suite, you might want to run a single test file, in which case, you can use the `--specs` option
```

Comment:
```suggestion
Since the test suite for the app is the biggest test suite, you might want to run a single test file, in which case, you can use the `--specs` option:
```